src/bindings/python/src/openvino/frontend/jax/utils.py

```diff
@@ -65,7 +66,7 @@ def get_type_from_py_type(value):
         return OVType.f32
     if isinstance(value, bool):
         return OVType.boolean
-    if isinstance(value, int):
+    if isinstance(value,(int, np.int64)):
```

Comment:
numpy type should be filtered out before calling `get_type_from_py_type` function

Reply:
Do you mean that I should convert all the elements with `np.int64` to `int` before calling `get_type_from_py_type`?

Reply:
No, I mean `get_type_from_py_type` is used only for python scalars. All numpy types should be processed with map `numpy_to_ov_type_map`